frontend/src/pages/LoginRegister/Register.js

```diff
@@ -10,26 +11,36 @@ function Register() {
     password: '',
     re_password: ''
   });
+
+  const [token, setToken] = useCookies(['mytoken']);
   const navigate = useNavigate();
   const handleChange = (event) =>
     setUserRegisterConfiguration({
       ...userRegisterConfiguration,
       [event.target.name]: event.target.value
     });
+
   const registerFormConfiguration = [
     { name: 'name', id: 'name', placeholder: 'Name', type: 'name' },
     { name: 'username', id: 'username', placeholder: 'Username', type: 'username' },
     { name: 'email', id: 'email', placeholder: 'Email', type: 'email' },
     { name: 'password', id: 'password', placeholder: 'Password', type: 'password' },
     { name: 're_password', id: 're_password', placeholder: 'Confirm Password', type: 'password' }
   ];
+
   const onRegister = () => {
     APIService.RegisterUser(userRegisterConfiguration)
       .then((resp) => console.log(resp))
       .catch((error) => console.log(error))
       .then(navigate('/login'));
   };
 
+  useEffect(() => {
+    if (token.mytoken) {
+      navigate('/');
+    }
+  }, [token]);
+
   return (
     <div className="w-full max-w-xs">
       {registerFormConfiguration.map(({ name, id, placeholder, type }) => (
```

Comment:
each mapped element here should have unique key, in `registerFormConfiguration` unique key is `id` so put `key={id}` in `<div>`